src/frontend/data/sideEffects/createLiveRegistration/index.ts

```diff
@@ -0,0 +1,22 @@
+import { API_ENDPOINT } from '../../../settings';
```

Comment:
Can you use absolute imports in place of ascending relative ones in this file please ?